docs/monitoring/cluster-alerts.asciidoc

```diff
@@ -17,6 +17,9 @@ different versions in the same cluster.
 versions running against the same {es} cluster.
 * Logstash Version Mismatch. You have Logstash nodes with different
 versions reporting stats to the same monitoring cluster.
+* {es} Nodes Changed. You have {es} nodes that were recently added or removed.
+* {es} License Expiration. The cluster's license is about to expire.
```

Comment:
Is it worthwhile to expand upon the details here, like how soon before expiration an alert will occur?

Reply:
Is that different from what's documented further below in this doc? https://github.com/elastic/kibana/pull/37495/files#diff-ce01bab25fe76ac78c8e4084c2d26dc0R29

Reply:
Nope, didn't notice that. Thanks!